django/utils/functional.py

```diff
@@ -9,7 +9,7 @@
 # CPython) is a type and its instances don't bind.
 def curry(_curried_func, *args, **kwargs):
     def _curried(*moreargs, **morekwargs):
-        return _curried_func(*(args + moreargs), **{**kwargs, **morekwargs})
+        return _curried_func(*args, *moreargs, **{**kwargs, **morekwargs})
```

Comment:
Note that we can only do this for the `args`/`moreargs` case, but not for `kwargs`/`morekwargs` unless the keys in `kwargs` and `morekwargs` do not intersect.